src/Service.Host/client/src/components/Workstations.js

```diff
@@ -73,14 +82,26 @@ function Workstations() {
         }
     ];
 
+    const hasPermission = utilities.getHref(workStationsResult?.[0], 'workstation.admin');
```

Comment:
This is to decide whether the user is permitted to create or not, right? Its generally not a good idea to rely existing resources (i.e. accessing the first item in the existing list to inspect its link relations) to decide whether to offer the client a create operation - for example what if there were none in existence, or they all got deleted and we wanted to start fresh? this code would make it impossible to ever create the first one since you're relying on their being a first one to access the create link.

Generally for link rels that dont pertain to an existing resource, e.g. rel: 'create' which pertains to the whole collection really, we make an /application-state request or similar that gets links without the requirement that the user has access to existing resources